### PR TITLE
Fix DBConfig not initialize issue in pfcwd

### PR DESCRIPTION
**Pull request by liuh-80**
#### What I did
Fix pfcwd connect DB with exception issue: https://github.com/Azure/sonic-buildimage/issues/11269

pfcwd implicit depends on InterfaceAliasConverter() to initialize DB config, however following PR change InterfaceAliasConverter() behavior to lazy initialize, then pfcwd failed when try connect to DB without initialize DB config:

https://github.com/Azure/sonic-utilities/pull/2183

#### How I did it
Load  DB config in pfcwd.

#### How to verify it
Pass all UT.

### CODE REVIEW

File: pfcwd/main.py [hunk 1]
```diff
@@ -110,6 +111,10 @@ def __init__(
     ):
         self.db = None
         self.config_db = None
+
+        # Load database config files
+        load_db_config()
+
```
Comment (SuvarnaMeenakshi) [inline pfcwd/main.py:117]: This did not help resolve the issue seen on multi-asic VS.
Adding at the cli entry point worked:
```
# Main entrypoint
@click.group()
def cli():
    """ SONiC PFC Watchdog """
    load_db_config()
```

Comment (qiluo-msft) [reply pfcwd/main.py:117]: How about moving to `__main__` section? It should be the entry point of this script.

Comment (SuvarnaMeenakshi) [reply pfcwd/main.py:117]: Adding it to __main__ did not help:
```
if __name__ == '__main__':
    load_db_config()
    cli = get_pfcwd_clis()
    cli()
```

Comment (SuvarnaMeenakshi) [reply pfcwd/main.py:117]: pfcwd start_default invokes: /usr/local/bin/pfcwd ; which invokes cli() in pfcwd.main.
```
admin@vlab-08:~$ vi /usr/local/bin/pfcwd 
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import re
import sys
from pfcwd.main import cli
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(cli()) 
```

Comment (qiluo-msft) [reply pfcwd/main.py:117]: This is cool finding. `__main__` section is not the entry point of this script.

Comment (SuvarnaMeenakshi) [reply pfcwd/main.py:117]: __main__ also invokes cli(), so having this change in cli() will ensure that invoking main also works.

Comment (liuh-80) [reply pfcwd/main.py]: Fixed, move load_db_config to cli()

File: pfcwd/main.py [hunk 2]
```diff
@@ -62,7 +63,12 @@
 @click.group()
 def cli():
     """ SONiC PFC Watchdog """
+    initialize_db_config()
 
+def initialize_db_config():
+    # Initialize db config here for UT coverage
+    load_db_config()
+    return
```
Comment (liuh-80) [inline pfcwd/main.py:71]: Our code coverage requirement is 80%, so need return here to pass coverage check.

Comment (qiluo-msft) [reply pfcwd/main.py:71]: Prefer keep code cleaner, and fix the test infra.

Comment (qiluo-msft) [reply pfcwd/main.py:71]: I am okay to override coverage check in this PR as long as there is an issue tracking the coverage issue.

Comment (liuh-80) [reply pfcwd/main.py:71]: Will improve code and try fix test infra later.

Comment (liuh-80) [reply pfcwd/main.py]: I checked the code coverage of some other script, the @click.group() of those script are all not covered, for example piceutil, so will try cover the cli() and if not work will change coverage rate to 50%

<html>
<body>




  |   | 69 |   | # This is our main entrypoint - the main 'psuutil' command
  | 1 | 70 |   | @click.group()
  |   | 71 |   | def cli():
  |   | 72 |   | """pcieutil - Command line utility for checking pci device"""
  | 0 | 73 |   | if os.geteuid() != 0:
  | 0 | 74 |   | click.echo("Root privileges are required for this operation")
  | 0 | 75 |   | sys.exit(1)
  |   | 76 |   |  
  |   | 77 |   | # Load platform-specific psuutil class
  | 0 | 78 |   | load_platform_pcieutil()
  |   | 79 |  

<br class="Apple-interchange-newline">
</body>
</html>

Comment (liuh-80) [reply pfcwd/main.py]: This seems a known issue in click: https://github.com/pallets/click/issues/267
Will try use subprocess.check_output to cover code in cli().

Comment (liuh-80) [reply pfcwd/main.py]: Revert the code coverage threshold change for force merge.

**Review by qiluo-msft — APPROVED** [review]

**Comment by rlhui** [comment]
@qiluo-msft , @SuvarnaMeenakshi  - does this one need to be in 202205  branch?

**Comment by SuvarnaMeenakshi** [comment]
> @qiluo-msft , @SuvarnaMeenakshi - does this one need to be in 202205 branch?

Not required in 202205 because the change that caused this issue https://github.com/Azure/sonic-utilities/pull/2183 is not there in 202205 branch

**Comment by ysmanman** [comment]
> > @qiluo-msft , @SuvarnaMeenakshi - does this one need to be in 202205 branch?
> 
> Not required in 202205 because the change that caused this issue #2183 is not there in 202205 branch

Just clarify, we ran into this issue with 202205 image. We filed https://github.com/sonic-net/sonic-buildimage/issues/12062 to track.

**Comment by liuh-80** [comment]
This PR need cherry pick to 202205 branch to fix https://github.com/sonic-net/sonic-buildimage/issues/12062

**Comment by gechiang** [comment]
@gechiang check if this can cherry picked into 20205 branch

**Comment by gechiang** [comment]
@liuh-80 , can you raise a new PR in 202205 branch to cherry pick this same change and I will help approve it and merge it so that we don't need to wait for 202205 branch owner to perform the cherry-pick.

**Comment by gechiang** [comment]
I removed the label request for 20205 branch because I manually created a pr to pick up thuis same change in 202205 here:
https://github.com/sonic-net/sonic-utilities/pull/2372

**Pull request merged**